CMakeLists.txt

```diff
@@ -255,13 +255,6 @@ if(USE_OPENCV)
   message(STATUS " OpenCV_LIBS=${OpenCV_LIBS}")
   message(STATUS "OpenCV found (${OpenCV_CONFIG_PATH})")
   add_definitions(-DMXNET_USE_OPENCV=1)
-  if(NOT MSVC)
```

Comment:
Please don't remove this.

Reply:
Should the case be empty then? Why are we putting linking flags in CXX_FLAGS? is giving me errors in Clang.